src/robot2017/src/robot_exec.cpp

```diff
@@ -1,17 +1,19 @@
 #include "robot_exec.h"
 #include "robot_msgs/Teleop.h"
-#include "motors.h"
+#include <vesc_bbb_uart/bldc.h>
 #include <ros/ros.h>
 #include <std_msgs/Int64.h>
 #include <string>
 
-Motor left(0);
-Motor right(1);
-
 RobotExec::RobotExec() : dead(true), autonomyActive(false),
                          leftRatio(0.0f), rightRatio(0.0f),
-                         driveLeft(0), driveRight(1)
+                         LeftDrive(LEFTDRIVE, CIM), 
+                         RightDrive(RIGHTDRIVE, CIM),
+                         Lift(LIFT, Alien_4260),
+                         Storage(STORAGE, Alien_4260),
+                         Bucket(BUCKET, Alien_4260)
 {
+    BLDC::init(0);
```

Comment:
Argument here should be file path the UART device - for now, use "/dev/ttyS0" until we know for sure